### PR TITLE
2707 - Flag metal

### LINKED ISSUE #2707: Put Metal work behind flags

### PR DESCRIPTION
**Pull request by jasonLaster**
Fixes #2707 

We're not going to support class and classify out of the gate so adding some flags

### CODE REVIEW

File: SpecRunner.html [hunk 1]
```diff
@@ -64,15 +64,22 @@
...
-    <script src="node_modules/backbone.radio/build/backbone.radio.js"></script>
-    <script src="node_modules/backbone-metal/dist/backbone-metal.js"></script>
-
 
     <script>
         window.Marionette = Backbone.Marionette = {};
         window.slice = Array.prototype.slice;
     </script>
+
+    <script src="src/features.js"></script>
```
Comment (jasonLaster) [inline SpecRunner.html]: features needs to be defined before metal.js

**Comment by paulfalgout** [comment]
lol.. Now it's jscs.  @jasonLaster don't use `.editorConfig`?

**Comment by jasonLaster** [comment]
Lol

File: docs/marionette.features.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Features
+
+Marionette Features are opt-in functionality. That you can enable by setting `Marionette.Features` in your app.
```
Comment (thejameskyle) [inline docs/marionette.features.md]:
```diff
- Marionette.Features
+ Marionette.FEATURES
```

Comment (paulfalgout) [inline docs/marionette.features.md]: ping @jasonLaster I think this is the last tweak here.

**Comment by ianmstew** [comment]
Oh boy, so we decided to go ahead and overwrite Backbone classes?

**Comment by paulfalgout** [comment]
Well I think we don't know exactly what's what with the metal stuff, but rather than revert  it's all just flagged.  ( https://github.com/marionettejs/backbone.marionette/issues/2637 )

**Comment by ianmstew** [comment]
Gotcha.  Yeah, given an experimental-feature flag, might as well experiment :)

**Comment by paulfalgout** [comment]
@jasonLaster one quick fix on this PR and it can get merged.

**Comment by jasonLaster** [comment]
My hanks Paul

**Comment by jasonLaster** [comment]
Lol autocorrect.  Will get to it soon

**Comment by paulfalgout** [comment]
Manually merged.

**Comment by jasonLaster** [comment]
Thanks!

**Pull request closed without merging**